projects/detr3d/mmdet3d_plugin/models/detectors/detr3d.py

```diff
@@ -0,0 +1,209 @@
+import os
+import time
+from typing import Dict, List, Optional, Sequence
+
+import numpy as np
+import torch
+from mmdet3d.models.detectors.mvx_two_stage import MVXTwoStageDetector
+from mmdet3d.registry import MODELS
+from mmdet3d.structures import Det3DDataSample
+from torch import Tensor
+
+from projects.detr3d.mmdet3d_plugin.models.utils.grid_mask import GridMask
+from mmdet3d.structures.bbox_3d.utils import get_lidar2img
+
+@MODELS.register_module()
+class Detr3D(MVXTwoStageDetector):
```

Comment:
Suggest adding `_version` in this class and overriding the `_load_from_state_dict` function by adding version loading. You can refer to https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/mmdet3d/models/detectors/mink_single_stage.py#L95-L137 and https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/mmdet3d/models/detectors/mink_single_stage.py#L42
In this way, we don't need the configs `*_test_old.py` and `detr3d_old.py`. We can decide whether to transform the bbox format in `predict` in `detr3d.py` according to the `_version`.
After this modification, remind that remove the `dev-1.x` and `test_old` words in the config names.

Reply:
Detr3D -> DETR3D